src/Components/Web.JS/src/DomWrapper.ts

```diff
@@ -22,7 +22,7 @@ function focus(element: HTMLOrSVGElement, preventScroll: boolean): void {
   }
 }
 
-function focusBySelector(selector: string, preventScroll: boolean): void {
+function focusBySelector(selector: string): Element | null {
```

Comment:
Was the boolean never used?

Reply:
Yeah, it was never used and we never provided a value for the argument anywhere, so it was always effectively `false`. Not that providing a value would have made a difference anyway (because it wasn't used).